Jenkinsfiles/README.md

```diff
@@ -0,0 +1,65 @@
+## Deployment procedures
+
+### Creating a new AMI
+
+See: [build ami](https://cloudbeesjenkins.cms.gov/dev-master/job/Blue%20Button/job/build%20ami/build?delay=0sec) Jenkins job
+
+Parameters:
+
+- BRANCH: Can be a git branch, git commit or git tag (e.g., develop) of the bluebutton-web-server repo
+- BUILD_BRANCH: Can be a git branch, git commit or git tag (e.g., master) of the bluebutton-deployment-repo used to orchestrate the AMI build
+- AMI_ID: The ID of the Gold Image AMI to be used as the foundation for the Blue Button AMI. View a list of Gold Image AMIs in the [AWS Console](https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Images:visibility=private-images;search=Gold%20Image;name=EAST-RH%207-4%20*;sort=desc:creationDate).
+- SUBNET_ID: The subnet where the AMI build EC2 instance should be launched. In lieu of a shared VPC for running build and deploy tasks, the recommended approach would be to use a management subnet for a given environment. For example, `subnet-81ecfbab`would be appropriate for building a `dev` AMI.
```

Comment:
Do you want the name of our subnet to be public? Also applies to some of the links here?

Reply:
I think the links are fine, but maybe I'll remove the subnet ID. Doesn't _have to_ be protected, but there's no reason to have it here. It's easy enough to find with access to AWS console.